pkg/controller/webnotifier/webnotifier.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package webnotifier
+
+import (
+	"fmt"
+
+	"github.com/hyperledger/aries-framework-go/pkg/common/log"
+	"github.com/hyperledger/aries-framework-go/pkg/controller/command"
+	"github.com/hyperledger/aries-framework-go/pkg/controller/rest"
+)
+
+var logger = log.New("aries-framework/webnotifier")
+
+// WebNotifier is a dispatcher capable of notifying multiple subscribers via HTTP Webhooks and WebSockets.
+type WebNotifier struct {
+	notifiers []command.Notifier
+	handlers  []rest.Handler
+}
+
+// New returns a new instance of a WebNotifier.
+func New(wsPath string, webhookURLs []string) *WebNotifier {
+	webhook := NewHTTPNotifier(webhookURLs)
+	ws := NewWSNotifier(wsPath)
+
+	n := WebNotifier{
+		notifiers: []command.Notifier{webhook, ws},
+		handlers:  ws.GetRESTHandlers(),
+	}
+
+	return &n
+}
+
+// Notify sends the given message to all of the subscribers.
+// If multiple errors are encountered, then the first one is returned.
+func (n *WebNotifier) Notify(topic string, message []byte) error {
+	var allErrs error
+
+	for _, notifier := range n.notifiers {
```

Comment:
at some point (multitenancy) we'll need to direct notifications to their subscribers

Reply:
yup - just following the existing code in this PR.